openlibrary/core/bookshelves_events.py

```diff
@@ -0,0 +1,113 @@
+from datetime import date, datetime
+from . import db
+
+
+class BookshelvesEvents:
```

Comment:
We can easily make this a subclass of [CommonExtras](https://github.com/internetarchive/openlibrary/blob/e0e7a87d6cf69fb0b9cfb0ee7596657af247e056/openlibrary/core/db.py#L22) if it's deemed useful to do so.

Reply:
Yeah we should, so that this table updates when work merges happen.